### DISCUSSION TITLE
Generate sql file for migrations

### DISCUSSION THREAD

**Discussion by SonnyMiel** [mikro-orm/mikro-orm#1520] [Q&A] [answered]
Hello, 

Would it be possible to generate migration file using sql format ? 

Instead of the usual js/ts extensions, I would prefer to have raw sql file so that I can use SQL Migration tools and not be dependant of any ORM to do my migrations.

Is it somehow possible using Mikro-orm ? 

Thanks !

**Comment by B4nan** [comment, marked as answer]
Nope, you cant have raw sql file, but the migration file pretty much consists only of raw sql, so not sure if that should matter.

If you dont want to be dependent on the ORM, dont use migrations from the ORM I guess? :] You could still use the SchemaGenerator to do the schema diffing (the hard part).
